commons/src/main/java/org/infinispan/commons/marshall/Ids.java

```diff
@@ -8,7 +8,7 @@
  */
 public interface Ids {
    int MURMURHASH_2 = 71;
-   int MURMURHASH_2_COMPAT = 72;
+   // 72 used to be MURMURHASH_2_COMPAT
```

Comment:
Is there any reason we are removing this?  I am kinda curious from an Id perspective as well?  Should we remove ```MURMURHASH_2``` as well then?